scripts/sg-ticket-link.coffee

```diff
@@ -28,5 +28,9 @@ module.exports = (robot) ->
     if isNaN(ticket_number)
       return
     
-    shotgun_url = process.env.HUBOT_SHOTGUN_URL
-    msg.send "#{shotgun_url}/detail/Ticket/#{ticket_number}"
+    if msg.message.match(/ZD#(\d+)/)
+      zendesk_url = process.env.HUBOT_ZENDESK_TICKET_URL
+      msg.send "#{zendesk_url}#{ticket_number}"
```

Comment:
I don't think we have the Zendesk script in this repo right now... I'm assuming there's an upstream somewhere I'm not looking at :p. As for the HipChat supported integration-- it doesn't currently support the Mac Desktop app.

As for whitespace-- what if we want to add other types in the future? shouldn't we be as specific as possible and not make assumptions?